src/providers/caching/route/model/cache-mode.ts

```diff
@@ -0,0 +1,5 @@
+export enum CacheMode {
```

Comment:
Can you include jsdoc on what these different modes are?

Reply:
Sure! do you have any example of how to document enums? I can search for it, but if you have the format handy that would be helpful